nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/ops/SDMath.java

```diff
@@ -2168,6 +2168,28 @@ public SDVariable step(String name, SDVariable in, double cutoff) {
         return updateVariableNameAndReference(ret, name);
     }
 
+    /**
+     * Standardize input variable along given axis
```

Comment:
Better javadoc would be good. Mainly clarify exactly what you mean by "standardize"